src/table/__integ__/tall-rows.test.ts

```diff
@@ -13,6 +13,7 @@ const setupTest = (testFn: (page: BasePageObject) => Promise<void>) => {
     const page = new BasePageObject(browser);
     await page.setWindowSize({ width: 900, height: 800 });
     await browser.url('#/light/table/tall-rows');
+    await page.waitForVisible(tableWrapper.findRows().toSelector());
```

Comment:
This is what we normally do in tests. Without this line the scroll on line 30 (`page.windowScrollTo({ top: 200 })`) caused no effect when run in React 18, thus making the assertion fail eventually.